addons/web/static/src/core/l10n/localization_service.js

```diff
@@ -6,23 +6,37 @@ import { strftimeToLuxonFormat } from "./dates";
 import { localization } from "./localization";
 import { translatedTerms, _t } from "./translation";
 
+const defaultLangParameters = {
```

Comment:
it's not obvious why we need default values. if this commit is merged, we need some explanations. is it for the frontend?

Reply:
It's for the login because there is no session_info in the login layout, but maybe I should add the frontend_session_info to the login layout instead